File: clients/printnanny-os/rust/src/systemd_manager_disable_units_reply.rs
// SystemdManagerDisableUnitsReply represents a SystemdManagerDisableUnitsReply model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct SystemdManagerDisableUnitsReply {
    #[serde(rename="request")]
    pub request: Box<crate::SystemdManagerUnitFilesRequest>,
    #[serde(rename="changes")]
    pub changes: Vec<crate::SystemdUnitChange>,
}

impl SystemdManagerDisableUnitsReply {
    pub fn new(request: crate::SystemdManagerUnitFilesRequest, changes: Vec<crate::SystemdUnitChange>) -> SystemdManagerDisableUnitsReply {
        SystemdManagerDisableUnitsReply {
            request: Box::new(request),
            changes,
        }
    }
}

File: clients/printnanny-os/typescript/src/SystemdManagerDisableUnitsReply.ts
import {SystemdManagerUnitFilesRequest} from './SystemdManagerUnitFilesRequest';
import {SystemdUnitChange} from './SystemdUnitChange';
interface SystemdManagerDisableUnitsReply {
  request: SystemdManagerUnitFilesRequest;
  changes: SystemdUnitChange[];
}
export { SystemdManagerDisableUnitsReply };